### PR TITLE
Set the pod_list in mdtf_gfdl.csh to include all the phase 2 PODs

### PR DESCRIPTION
**Pull request by Wen-hao-Dong**
**Description**
Set the pod_list in mdtf_gfdl.csh to include all the phase 2 PODs
i.e. set pod_list = '"Wheeler_Kiladis","EOF_500hPa","convective_transition_diag","MJO_suite","MJO_teleconnection","MJO_prop_amp","precip_diurnal_cycle","SM_ET_coupling"'

Associated issue # (replace this phrase and parentheses with the issue number)  

**How Has This Been Tested?**
Please describe the tests that you ran to verify your changes in enough detail that  
someone can reproduce them. Include any relevant details for your test configuration  
such as the Python version, package versions, expected POD wallclock time, and the   
operating system(s) you ran your tests on.

**Checklist:**
- [x] I have reviewed my own code to ensure that if follows the [POD development guidelines](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/dev_guidelines.html)
- [x] My branch is up-to-date with the NOAA-GFDL main branch, and all merge conflicts are resolved
- [ ] The scripts are written in Python 3.7 or above (preferred; required if funded by a CPO grant), NCL, or R
- [x] All of my scripts are in the diagnostics/[POD short name] subdirectory, and include a main_driver script, template html, and settings.jsonc file
- [ ] I have made corresponding changes to the documentation in the POD's doc/ subdirectory
- [ ] I have requested that the framework developers add packages required by my POD to the python3, NCL, or R environment yaml file if necessary, and my environment builds with `conda_env_setup.sh` 
- [ ] I have added any necessary data to input_data/obs_data/[pod short name] and/or input_data/model/[pod short name]
- [ ] My code is portable; it uses [MDTF environment variables](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_envvars.html), and does not contain hard-coded file or directory paths
- [ ] I have provided the code to generate digested data files from raw data files
- [ ] Each digested data file generated by the script contains numerical data (no figures), and is 3 GB or less in size
- [ ] The repository contains no extra test scripts or data files

### CODE REVIEW

File: sites/NOAA_GFDL/default_gfdl.jsonc [hunk 1]
```diff
@@ -22,14 +22,19 @@
...
+          //"Wheeler_Kiladis",
+          //"EOF_500hPa",
+          //"convective_transition_diag",
+          //"MJO_suite",
+          //"MJO_teleconnection",
           //"MJO_prop_amp",
-          "precip_diurnal_cycle"
-         // "SM_ET_coupling"
+          //"precip_diurnal_cycle"
+          //"SM_ET_coupling"
```
Comment (jkrasting) [inline sites/NOAA_GFDL/default_gfdl.jsonc:32]: Should all of the PODs be turned on?

**Review by wrongkindofdoctor — CHANGES_REQUESTED** [review]

File: sites/NOAA_GFDL/default_gfdl.jsonc (hunk 1 above)
Comment (Wen-hao-Dong) [reply sites/NOAA_GFDL/default_gfdl.jsonc:32]: The new PODs have only been tested in an offline mode and we have seen several problems. Not sure if we should include them into the frepp framework at this time

File: my_gfdl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (Wen-hao-Dong) [reply my_gfdl:1]: Got it, but the mdtf_slurm is also created for personal use, should we also remove it or modify it to make it generalized?

[the remark this answers is not in the document]
Comment (wrongkindofdoctor) [reply my_gfdl:1]: @Wen-hao-Dong My bad--I thought I was looking at mdtf_gfdl.csh when I was commenting. Yes, remove mdtf_slurm from the PR (and add it to your .gitignore file if you want) and don't worry about generalizing it.

File: sites/NOAA_GFDL/default_gfdl.jsonc [hunk 3]
```diff
@@ -51,7 +56,8 @@
   "WORKING_DIR": "$MDTF_TMPDIR/wkdir",
 
   // Directory to write output files. Defaults to working directory if blank.
-  "OUTPUT_DIR": "$MDTF_TMPDIR/mdtf_out",
+  //"OUTPUT_DIR": "$MDTF_TMPDIR/mdtf_out",
```
[the remark this answers is not in the document]
Comment (Wen-hao-Dong) [reply sites/NOAA_GFDL/default_gfdl.jsonc]: Done, please help double-check, I have also turned on all the PODs in sites/NOAA_GFDL/default_gfdl.jsonc

File: sites/NOAA_GFDL/mdtf_gfdl.csh [hunk 4]
```diff
@@ -31,7 +31,7 @@ set model = "modelname"
 # Please enter the POD lists below. This gets filled in to the MDTF input json automatically if using frepp
 #this is tested with one POD only at this time
 
-set pod_list = '"EOF_500hPa"'
+set pod_list = '"Wheeler_Kiladis","EOF_500hPa","convective_transition_diag","MJO_suite","MJO_teleconnection","MJO_prop_amp","precip_diurnal_cycle","SM_ET_coupling"'
```
Comment (aradhakrishnanGFDL) [inline sites/NOAA_GFDL/mdtf_gfdl.csh:34]: @Wen-hao-Dong Thanks! A comment for the users regarding this line and the user flexibility to change the PODs based on this super set would be helpful. Everything else looks great with the others' review and resolutions.

Comment (aradhakrishnanGFDL) [inline sites/NOAA_GFDL/mdtf_gfdl.csh:34]:
```suggestion
#pod_list contains list of all PODS in the framework for reference. Please edit the pod_list as needed for your simulation. 
set pod_list = '"Wheeler_Kiladis","EOF_500hPa","convective_transition_diag","MJO_suite","MJO_teleconnection","MJO_prop_amp","precip_diurnal_cycle","SM_ET_coupling"' 
```

Comment (Wen-hao-Dong) [reply sites/NOAA_GFDL/mdtf_gfdl.csh:34]: @aradhakrishnanGFDL  Thanks for the suggestions, I have clicked the 'commit suggestion'

**Pull request merged**